### ISSUE TITLE
Update code for getting the last jsc built product on Linux

### ISSUE THREAD

**Issue by clopez** [GoogleChromeLabs/jsvu#98]
The built products available at https://webkitgtk.org/jsc-built-products were following the pattern of
```
- version.zip
- version.sha256sum
```
I changed that in https://bugs.webkit.org/show_bug.cgi?id=215266 and now the pattern is 
```
- version.zip
- version.zip.sha256sum
```

So the code in engines/javascriptcore/get-latest-version.js has to be updated accordingly.

BTW, instead of sorting the apache/http directory view by last modified and getting the last sha256sum file to get the last version you can now directly query the file https://webkitgtk.org/jsc-built-products/x86_64/release/LAST-IS . That file is guaranteed to always have the filename of the last built-product in that directory, and the built-product is guaranteed to have been completely written since the [LAST-IS file is updated after uploading the built-product and the sha256sum file](https://trac.webkit.org/browser/webkit/trunk/Tools/Scripts/generate-bundle?rev=265606#L547). It also has the advantage that if we change the file name pattern in the future the LAST-IS file will still contain the right filename to be downloaded.

**Comment by mathiasbynens** [comment]
Thanks!

https://webkitgtk.org/jsc-built-products/x86_32/release/ is empty, and https://webkitgtk.org/jsc-built-products/x86_32/release/LAST-IS does not exist — is that intentional? Are there no more linux32 JSC builds?

**Comment by clopez** [comment]
> https://webkitgtk.org/jsc-built-products/x86_32/release/ is empty, and https://webkitgtk.org/jsc-built-products/x86_32/release/LAST-IS does not exist — is that intentional? Are there no more linux32 JSC builds?

It has been already a while (more than 1 year) that we don't have any i686 JSC or WebKitGTK builder/tester at build.webkit.org
So we are not providing Linux x86_32 JSC builds anymore.

**Comment by mathiasbynens** [comment]
Thanks. The changes have landed and are available in `jsvu@1.11.4`.
